lib/api/controllers/admin.js

```diff
@@ -172,8 +172,8 @@ class AdminController extends NativeController {
 
   loadSecurities (request) {
     const securities = this.getBody(request);
-
-    const promise = this.kuzzle.janitor.loadSecurities(securities, request.context.user._id);
+    const user = this.getUser(request);
+    const promise = this.kuzzle.janitor.loadSecurities(securities, { user });
```

Comment:
`janitor.loadSecurities` seems to expect a User object, not an anonymous `{user: <User>}` object